app/views/ProfileView/index.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { View, ScrollView, Keyboard } from 'react-native';
+import {
+	View,
+	ScrollView,
+	Keyboard,
+	Alert
```

Comment:
Remember to remove these changes when you switch from `Alert` to `showErrorAlert`.